src/combinatorics/colex.lean

```diff
@@ -65,7 +64,7 @@ def finset.to_colex {α} (s : finset α) : finset.colex α := s
 
 @[simp]
 lemma colex.eq_iff (A B : finset α) :
-  A.to_colex = B.to_colex ↔ A = B := by refl
+  A.to_colex = B.to_colex ↔ A = B := iff.rfl
 
 /--
 `A` is less than `B` in the colex ordering if the largest thing that's not in both sets is in B.
```

Comment:
Is the `▵` on the line below the `symm_diff` operator? Can we replace it with the big delta we use for that operator instead?

Reply:
Sure - I think this docstring was before `symm_diff` was a thing in mathlib! Thanks for the catch